modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Manage/User/UserForm.tsx

```diff
@@ -280,6 +290,34 @@ const UserForm = () => {
 					</ClayLayout.Col>
 				</ClayLayout.Row>
 
+				<Form.Divider />
+
+				{handleDelete && (
+					<ClayLayout.Row className="mb-6" justify="start">
+						<ClayLayout.Col size={3} sm={12} xl={3}>
+							<h5 className="font-weight-normal">
+								{i18n.translate('delete-user')}
+							</h5>
+						</ClayLayout.Col>
+
+						<ClayLayout.Col size={3} sm={12} xl={3}>
+							<ClayForm.Group className="form-group-sm">
+								<ClayButton
+									displayType="danger"
+									onClick={() =>
+										liferayUserAccountsImpl
+											.remove(Number(userAccount.id))
+											.then(() => onSave())
```

Comment:
```suggestion
											.remove(userAccount.id)
											.then(() => onSave())
```

Reply:
I think we shouldn't use **onSave**, would be better do redirect to manage-users page